sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -33,11 +33,137 @@
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
+/**
+ * Table of forward branch Control Flow Integrity (CFI) counters.
+ *
+ * Columns: Name, Initital Value.
+ *
+ * Each counter is indexed by Name. The Initial Value is used to initialize the
+ * counters with unique values with a good hamming distance. The values are
+ * restricted to 12-bit to be able use immediate load instructions.
```

Comment:
IIRC, immediates are 12-bits but sign-extended. We should check the disassembly but we probably want to use 11-bits here (like we do for hardened_bool_t).

Reply:
Updated the values to 11-bit. I will post the disassembly to check this.